### ISSUE TITLE
Cannot create a new user

### ISSUE THREAD

**Issue by hamann** [epiccastle/spire#119]
```sh
❯ spire --version
Version: 0.1.0-alpha.15
```

Host: Ubuntu 18.04.4 LTS

```txt                                                                             <<<
src/deploy-user.clj:4 (user :present {:name "deploy" :group "deploy", :groups ["docker"], :shell "/usr/bin/zsh"}) hamann@xxx
hamann@xxx failed! exit:1
--stdout--
--stderr--
Exception in thread "main" groups: ‘deploy’: no such user
----------
clojure.lang.ExceptionInfo: module failed [at src/deploy-user.clj, line 3, column 1] {:out-lines [""], :exit 1, :err "groups: ‘deploy’: no such user\n", :type :sci/error, :column 1, :line 3, :out "", :result :failed, :message "modu
e failed [at src/deploy-user.clj, line 3, column 1]"}
        at sci.impl.utils$rethrow_with_location_of_node.invokeStatic(utils.cljc:72)
        at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:573)
        at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:613)
        at sci.impl.interpreter$eval_form.invokeStatic(interpreter.cljc:645)
        at sci.impl.interpreter$eval_string_STAR_.invokeStatic(interpreter.cljc:658)
        at sci.core$eval_string_STAR_.invokeStatic(core.cljc:189)
        at spire.eval$evaluate.invokeStatic(eval.clj:87)
        at spire.core$_main.invokeStatic(core.clj:80)
        at spire.core$_main.doInvoke(core.clj:57)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at spire.core.main(Unknown Source)
Caused by: clojure.lang.ExceptionInfo: module failed {:exit 1, :out "", :err "groups: ‘deploy’: no such user\n", :out-lines [""], :result :failed}
        at spire.module.user$user_STAR_.invokeStatic(user.clj:84)
        at spire.module.user$user_STAR_.doInvoke(user.clj:84)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at sci.impl.interpreter$fn_call.invokeStatic(interpreter.cljc:522)
        at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:573)
        at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:613)
        at sci.impl.interpreter$interpret.invoke(interpreter.cljc:590)
        at sci.impl.interpreter$eval_do_STAR_$fn__5649.invoke(interpreter.cljc:479)
        at sci.impl.interpreter$eval_do_STAR_.invokeStatic(interpreter.cljc:479)
        at sci.impl.interpreter$eval_do.invokeStatic(interpreter.cljc:488)
        at sci.impl.interpreter$eval_special_call.invokeStatic(interpreter.cljc:555)
        at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:564)
        at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:613)
        at sci.impl.interpreter$interpret.invoke(interpreter.cljc:590)
        at sci.impl.interpreter$eval_let$fn__5469.invoke(interpreter.cljc:76)
        at sci.impl.interpreter$eval_let.invokeStatic(interpreter.cljc:64)
        at sci.impl.interpreter$eval_let.doInvoke(interpreter.cljc:61)
        at clojure.lang.RestFn.applyTo(RestFn.java:142)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at sci.impl.interpreter$eval_special_call.invokeStatic(interpreter.cljc:536)
        at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:564)
        at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:613)
        at sci.impl.interpreter$interpret.invoke(interpreter.cljc:590)
        at sci.impl.interpreter$eval_do_STAR_$fn__5649.invoke(interpreter.cljc:479)
        at sci.impl.interpreter$eval_do_STAR_.invokeStatic(interpreter.cljc:479)
        at sci.impl.interpreter$eval_do.invokeStatic(interpreter.cljc:488)
        at sci.impl.interpreter$eval_special_call.invokeStatic(interpreter.cljc:555)
        at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:564)
        at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:613)
        at sci.impl.interpreter$interpret.invoke(interpreter.cljc:590)
        at sci.impl.interpreter$eval_try.invokeStatic(interpreter.cljc:291)
        at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:613)
        at sci.impl.interpreter$interpret.invoke(interpreter.cljc:590)
        at sci.impl.interpreter$eval_do_STAR_$fn__5649.invoke(interpreter.cljc:479)
        at sci.impl.interpreter$eval_do_STAR_.invokeStatic(interpreter.cljc:479)
        at sci.impl.interpreter$eval_do.invokeStatic(interpreter.cljc:488)
        at sci.impl.interpreter$eval_special_call.invokeStatic(interpreter.cljc:555)
        at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:564)
        at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:613)
        at sci.impl.interpreter$interpret.invoke(interpreter.cljc:590)
        at sci.impl.interpreter$eval_do_STAR_$fn__5649.invoke(interpreter.cljc:479)
        at sci.impl.interpreter$eval_do_STAR_.invokeStatic(interpreter.cljc:479)
        at sci.impl.interpreter$eval_do.invokeStatic(interpreter.cljc:488)
        at sci.impl.interpreter$eval_special_call.invokeStatic(interpreter.cljc:555)
        at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:564)
        at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:613)
        at sci.impl.interpreter$interpret.invoke(interpreter.cljc:590)
        at sci.impl.interpreter$eval_let$fn__5469.invoke(interpreter.cljc:76)
        at sci.impl.interpreter$eval_let.invokeStatic(interpreter.cljc:64)
        at sci.impl.interpreter$eval_let.doInvoke(interpreter.cljc:61)
        at clojure.lang.RestFn.applyTo(RestFn.java:142)
        at clojure.core$apply.invokeStatic(core.clj:667)
```

**Comment by retrogradeorbit** [comment]
@hamann versions with fix for you to test here:
macos: https://1307-221739364-gh.circle-artifacts.com/0/build/spire-0.1.0-alpha.16-macos-amd64.zip
linux: https://epiccastle.io/spire/builds/spire-0.1.0-alpha.16-linux-amd64.tgz

**Comment by hamann** [comment]
@retrogradeorbit Works!

**Comment by retrogradeorbit** [comment]
I'll just leave this open until that version is released. Thanks for verifying!

**Comment by hamann** [comment]
@retrogradeorbit we again run into this issue and your build isn't available anymore :(

**Comment by retrogradeorbit** [comment]
released in v0.1.0-alpha.17
